staging/kubefed/charts/controllermanager/templates/webhook.yaml

```diff
@@ -137,3 +150,49 @@ type: kubernetes.io/tls
 stringData:
   tls.crt: {{ $cert.Cert | quote }}
   tls.key: {{ $cert.Key | quote }}
+{{- else }}
+---
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Issuer
+metadata:
+  name: {{ .Release.Name }}-ca-issuer
+spec:
+  selfSigned: {}
+---
+# Generate a CA Certificate used to sign certificates for the webhook
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: {{ .Release.Name}}-root-certificate
+spec:
+  secretName: {{ .Release.Name }}-root-ca
+  duration: 43800h # 5y
```

Comment:
would we potentially want to lower this? Keeping this long could result in unintended consequences for the end-user, theoretically the following scenario:

1. PK is compromised by a bad actor
2. bad actor generates a bunch of bad certs
3. any client that trusts the CA (and maybe doesn't get a memo about the breach) continues to do so for up to 5 years

By declaring shorter CA's we can help to reduce some risks in our end-users environments and help make breach mitigations a bit more manageable by having very conservative defaults.

Let me know your thoughts on this :thinking: